sunpy/instr/fermi.py

```diff
@@ -0,0 +1,373 @@
+from __future__ import absolute_import
+from __future__ import division
+
+import numpy as np
+import copy
+import math
+import urlparse
+import urllib, urllib2
+import tempfile
+import datetime
+import os
+import matplotlib.pyplot as plt
+
+from bs4 import BeautifulSoup
+import astropy.units as u
+from astropy.utils.compat.odict import OrderedDict
+from astropy.coordinates import Angle, Longitude, Latitude
+
+from sunpy.time import parse_time, TimeRange
+from sunpy import sun
+from sunpy.io.fits import fits
```

Comment:
Using this convention for `fits` is a little misleading as the idiom for astropy is `from astropy.io import fits`.

On top of this, the file type independent wrappers in `sunpy.io`, i.e. `sunpy.io.read()` should do what you need (if they don't it is probably a bug) it would be better to use those if you can.